src/category_theory/monoidal/rigid/basic.lean

```diff
@@ -227,6 +235,298 @@ begin
     right_unitor_naturality_assoc, ←unitors_equal, ←category.assoc, ←category.assoc], simp
 end
 
+/--
+Given an exact pairing on `Y Y'`,
+we get a bijection on hom-sets `(Y' ⊗ X ⟶ Z) ≃ (X ⟶ Y ⊗ Z)`
+by "pulling the string on the left" up or down.
+
+This gives the adjunction `tensor_left_adjunction Y Y' : tensor_left Y' ⊣ tensor_left Y`.
+
+This adjunction is often referred to as "Frobenius reciprocity" in the
+fusion categories / planar algebras / subfactors literature.
+-/
+def tensor_left_hom_equiv (X Y Y' Z : C) [exact_pairing Y Y'] :
```

Comment:
Apologies for not seeing this sooner, but what is the relationship between this and https://leanprover-community.github.io/mathlib_docs/category_theory/closed/functor.html#category_theory.frobenius_morphism?